linkml_model/model/schema/meta.yaml

```diff
@@ -110,9 +110,22 @@ slots:
     in_subset:
       - owl
 
+  type:
+    range: type_enum
+    description: >-
+      The relatedness of an alias to the element it is aliasing.  type can be: narrow, broad, exact,
```

Comment:
Instead of just `narrow` and `broad`, do you want `is narrower than` and `is broader than` ? It may make it clearer.